.github/workflows/cross.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Cross-Build Checks
+
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v**'
```

Comment:
I think I missed this one, whats the purpose of testing this on tags?

Reply:
Copy-pasta...